plugins/aws-appmesh/plugin/commands.go

```diff
@@ -214,29 +215,10 @@ func (plugin *Plugin) setupIngressRules(
 	if config.ProxyIngressPort == "" || len(config.AppPorts) == 0 {
 		return nil
 	}
-
-	err := iptable.NewChain("nat", ingressChain)
-	if err != nil {
-		return err
+	var interceptPortToListenerPorts = map[string]string{
+		config.ProxyIngressPort: config.AppPorts,
```

Comment:
So, this is a mapping between the port to be redirected -> comma separate list of listener ports. In AppMesh, this would be 15001 -> <list of App Ports> whereas in Service Connect, this would be ephemeral port/listener port -> App port/intercept port. Renamed this to `redirectPortToListenerPorts` to make this clear.